lib/environment.ts

```diff
@@ -1,37 +1,60 @@
-import fs from "fs";
+import { promises as fsPromises } from "fs";
```

Comment:
@magul you can use `fs-extra` as it defaults to promises and comes with some additional, handy functions.